src/main/java/com/mraof/minestuck/computer/theme/ComputerThemeManager.java

```diff
@@ -0,0 +1,99 @@
+package com.mraof.minestuck.computer.theme;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.packs.resources.PreparableReloadListener;
+import net.minecraft.server.packs.resources.ResourceManager;
+import net.minecraft.server.packs.resources.SimpleJsonResourceReloadListener;
+import net.minecraft.util.profiling.ProfilerFiller;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.RegisterClientReloadListenersEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.*;
+
+/**
+ * Helps acquire json files in assets/minestuck/minestuck/computer_themes/
+ * ComputerThemes are data driven resource pack files that determine a computers wallpaper and text color.
+ */
+@Mod.EventBusSubscriber(modid = Minestuck.MOD_ID, bus = Mod.EventBusSubscriber.Bus.MOD, value = Dist.CLIENT)
+public class ComputerThemeManager extends SimpleJsonResourceReloadListener implements PreparableReloadListener
```

Comment:
During #512 I had an idea to separate the reload listener "loader" from the data fields and query functions, to instead make it an inner class (see `LandTypeExtensions.Loader`). I think the same pattern could be good here.